core/src/main/java/hudson/model/AdministrativeMonitor.java

```diff
@@ -103,6 +101,11 @@ protected AdministrativeMonitor(String id) {
         this.id = id;
     }
 
+    public HttpResponse doAct(@QueryParameter String no) throws IOException{
+        //send the traffic to  /manage by default, can be overridden for nuanced checks.
+        return HttpResponses.redirectViaContextPath("/manage");
+    };
```

Comment:
Makes no sense to pull up even if it's a common pattern as the UI triggering this needs to be written each time anyway.